.bsp/sbt.json

```diff
@@ -1,18 +1 @@
-{
```

Comment:
we should probably add this to .gitignore so that it's not maintained in git

Reply:
.bsp/ is in .gitignore. Does that not include the sbt.json file inside?